### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Trailblazer007** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#42] [Outreachy 2022] [answered]
Hello everyone, I'm Seun from Nigeria. I'm new here and I'd want to be on this project. I've clicked some links and all, but I'm still a bit confused as to my way around and if I've actually been made a part of this project.
I'd really appreciate some help. Thanks.

**Comment by BatoolMM** [comment, marked as answer]
Welcome, @Trailblazer007 to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Some of the tasks in the issue have been addressed - you are also welcome to review open PR!

Let me know if you are still confused!

Feel free to comment under the issue, if you have any questions!
